src/FwdState.h

```diff
@@ -195,7 +196,7 @@ class FwdState: public RefCountable, public PeerSelectionInitiator
 
     HappyConnOpenerPointer connOpener; ///< current connection opening job
     ResolvedPeersPointer destinations; ///< paths for forwarding the request
-    Comm::ConnectionPointer serverConn; ///< a successfully opened connection to a server.
+    PeerConnectionPointer serverConn; ///< a successfully opened connection to a server
```

Comment:
This PR abuses `serverConn` to also store a half-baked returnable peer connection. I do not like that abuse because it obscures the boundary between "somebody else is opening our future connection for us" and "FwdState co-owns the connection and, hence, needs to maintain a closure callback" states. However, adding a dedicated PeerConnectionPointer field (to prevent this abuse) may also be confusing for other reasons.

Let's try to fix documentation:

```suggestion
    /// a peer selection result, a successfully opened connection to a server, or both
    PeerConnectionPointer serverConn;
```

Reply:
Done.